index.html

```diff
@@ -0,0 +1,174 @@
+                    Pennsylvania two years later, where he received a bachelor's degree in economics and physics, then
+                    moved to California in 1995 to attend Stanford University but decided instead to pursue a business
+                    career, co-founding the web software company Zip2 with his brother Kimbal. The startup was acquired
+                    by Compaq for $307 million in 1999. The same year, Musk co-founded online bank X.com, which merged
+                    with Confinity in 2000 to form PayPal. The company was bought by eBay in 2002 for $1.5 billion.
+
+                    In 2002, Musk founded SpaceX, an aerospace manufacturer and space transport services company, of
+                    which he is CEO and CTO. In 2004, he joined electric vehicle manufacturer Tesla Motors, Inc. (now
+                    Tesla, Inc.) as chairman and product architect, becoming its CEO in 2008. In 2006, he helped create
+                    SolarCity, a solar energy services company that was later acquired by Tesla and became Tesla Energy.
+                    In 2015, he co-founded OpenAI, a nonprofit research company that promotes friendly artificial
+                    intelligence. In 2016, he co-founded Neuralink, a neurotechnology company focused on developing
+                    brain–computer interfaces, and founded The Boring Company, a tunnel construction company. Musk has
+                    proposed the Hyperloop, a high-speed vactrain transportation system.</p>
+            </section>
+            <section class="intro">
+                <h3 class="vision">Tribute to Musk!</h3>
+                <p>
+                    Musk’s early interests in philosophy, science fiction, and fantasy novels are reflected in his sense
+                    of idealism and concern with human progress—and in his business career. He works in the areas that
+                    he has identified as crucial to our future, specifically the Internet, the transition to renewable
+                    energy sources, and space colonization. He has defied critics, disrupted industries, and made
+                    advances in all three of these frontiers, via his creation of PayPal, Tesla Motors, SolarCity, and
+                    SpaceX—game-changers all, despite their sometimes rocky performance.
+                </p>
+                <p>
+                    Elon Musk has a great vision for the future, perhaps the most unique one someone ever might have
+                    come up with. He knows what his doing. He's like an example of real-life Bruce Wayne.
+                </p>
+
+
+            </section>
+
+        </article>
+        <p id="footer">Click here to know More About
+            <a href="https://en.wikipedia.org/wiki/Elon_Musk" id="tribute-link" target="_blank"> Elon Musk.</a> Don't be shy ;)
+        </p>

```

Comment:
Adding this section inside footer tag would make more sense instead of using p tag with class as a footer as this creates confusion if someone else is reading your code